pkg/apis/serving/v1beta1/inference_service_status.go

```diff
@@ -34,7 +34,8 @@ type InferenceServiceStatus struct {
 	// - PredictorReady: predictor readiness condition; <br/>
 	// - TransformerReady: transformer readiness condition; <br/>
 	// - ExplainerReady: explainer readiness condition; <br/>
-	// - RoutesReady: aggregated routing condition; <br/>
+	// - RoutesReady: aggregated routing condition, i.e. endpoint readiness condition; <br/>
+	// - LatestDeploymentReady: aggregated configuration condition, i.e. latest deployment readiness condition; <br/>
```

Comment:
Need to add a note that these two conditions are only applicable for serverless mode.

Reply:
Actually the doc is not generated on CRD, we need a PR to the website to generate the [API doc](https://kserve.github.io/website/0.10/reference/api/) using the [script](https://github.com/kserve/website/blob/main/docs/hack/gen-api-reference-docs.sh).